docs/streams/developer-guide.html

```diff
@@ -1723,8 +1726,7 @@
         KTable&lt;String, Integer&gt; aggregated = groupedStream.aggregate(
             () -> 0, /* initializer */
             (aggKey, newValue, aggValue) -> aggValue + newValue, /* adder */
-            Serdes.Integer(), /* serde for aggregate value */
```

Comment:
Why remove the Serde?

Reply:
if the default serdes are set to String, Integer, then we don't need the `Serde`. Otherwise it will need to be `Materialized`, i was trying to keep it brief. I'll add the defaults